eng/WpfArcadeSdk/tools/CodeGen/DesignTimeTextTemplating.targets

```diff
@@ -0,0 +1,98 @@
+<Project>
+  <ItemGroup>
+    <Service Include="{508349b6-6b84-4df5-91f0-309beebad82d}" />
+  </ItemGroup>
+  <!-- Including the Microsoft.TextTemplating.targets requires us to manually import Sdk.Targets because
+       it needs to be imported after Sdk.targets. This causes the BuildDependsOn variable,
+       which is modified by the T4 targets, to be overwritten, so the TransformAll target 
+       doesn’t run before the Build target. -->
+  <!-- <Import Project="Sdk.targets" Sdk="Microsoft.NET.Sdk" /> -->
+  <PropertyGroup>
+    <!-- Get the Visual Studio version: -->
+    <VisualStudioVersion Condition="'$(VisualStudioVersion)' == ''">16.0</VisualStudioVersion>
+    <!-- Keep the next element all on one line: -->
+    <VSToolsPath Condition="'$(VSToolsPath)' == ''">$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)</VSToolsPath>
+    <!-- This is what will cause the templates to be transformed when the project is built (default is false) -->
+    <TransformOnBuild>true</TransformOnBuild>
+    <!-- Set to true to force overwriting of read-only output files, e.g. if they're not checked out (default is false) -->
+    <OverwriteReadOnlyOutputFiles>true</OverwriteReadOnlyOutputFiles>
+    <!-- Set to false to transform files even if the output appears to be up-to-date (default is true)  -->
+    <TransformOutOfDateOnly>false</TransformOutOfDateOnly>
+    <NewtonsoftVersion>12.0.2-beta1</NewtonsoftVersion>
```

Comment:
`<NewtonsoftVersion>12.0.2-beta1</NewtonsoftVersion>` should be rewritten as something like `<NewtonsoftVersion>$(NewtonsoftJsonVersion)</NewtonsoftVersion>`. where `$(NewtonsoftJsonVersion)` is defined in `eng\Version.props` because of a DARC-created dependency.

Reply:
does darc work for non-dotnet built assemblies? i had purposefully isolated this from the rest of the project because i didn't want to start introducing this into the product unnecessarily

Reply:
Even if DARC doesn't work for dependency updates (you'll have to test and identify whether a dependency is meaningful for this package), please don't hard-code the version here unless you have a dependency on a specific version. Instead, add the version into eng\Versions.props and consume an MSBuild variable here. You just won't have a DARC dependency, but that's ok.

Reply:
i converted to using xml

Reply:
yeah i hear ya. i moved over to xml though so i'm just removing this